assets/css/base/_base.scss

```diff
@@ -1 +1,52 @@
 // Base styles.
+
+// Typography
+h1 {
+	font-size: 4em ;
+	font-weight: semi-bold ;
+}
+h2 {
+	font-size: 3em ;
+	font-weight: medium ;
+}
+h3 {
+	font-size: 1.5em ;
+	font-weight: bold ;
+}
+p {
+	font-size: 1em ;
+	font-weight: regular ;
+}
+
+// Links and Buttons
+a {
+	color: #6895c6;
+	font-weight: bold;
+	text-decoration: none;
+}
+a:hover,
+a:focus {
+	background-color: #FADC90;
+}
+
+button {
```

Comment:
The button does not appear as designed.

Figma:

<img width="194" alt="Screen Shot 2020-03-10 at 6 47 36 PM" src="https://user-images.githubusercontent.com/605361/76371343-a0ab8800-62ff-11ea-829b-3b81db7911cc.png">

Chrome: 

<img width="64" alt="Screen Shot 2020-03-10 at 6 48 23 PM" src="https://user-images.githubusercontent.com/605361/76371412-ccc70900-62ff-11ea-89a3-d0a5b950ed2a.png">

Safari:

<img width="64" alt="Screen Shot 2020-03-10 at 6 48 37 PM" src="https://user-images.githubusercontent.com/605361/76371418-d2245380-62ff-11ea-87c3-702e06c22ead.png">

Firefox:

<img width="64" alt="Screen Shot 2020-03-10 at 6 48 42 PM" src="https://user-images.githubusercontent.com/605361/76371421-d81a3480-62ff-11ea-81c0-427d493a68c0.png">

Needs styling for font size, weight, and line height, padding, and height.